roles/cloud-bootstrap/tasks/create-users-projects.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Create bonnyci project
+  os_project:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci-admin group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci-admin
+
+- name: Add all users to bonnyci group
+  os_user_group:
+    cloud: "{{ cloud }}"
+    group: bonnyci
+    user: "{{ item.username }}"
+  with_items: users
+
+- name: Add all bonnyci group members to bonnyci project
+  os_user_role:
+    cloud: "{{ cloud }}"
+    role: member
+    group: bonnyci
+    project: bonnyci
+
+- name: Add admin users to bonnyci-admin group
+  os_user_group:
+    cloud: "{{ cloud }}"
+    group: bonnyci-admin
+    user: "{{ item.username }}"
+  with_items: users
+  when: item.isadmin == True
```

Comment:
It's preferable to use the `bool` filter to check for truthiness. Also, since this attribute might only be set WHEN a user is an admin, we have to handle the default case where it is not defined:

`when: item.isadmin | default('False') | bool`